content/implementations.md

```diff
@@ -0,0 +1,87 @@
+---
+doc-note-type: draft
+layout: standalone_resource
+title: "Accessibility Test Tools & Methodologies"
+permalink: /standards-guidelines/act/implementations/
+ref: /standards-guidelines/act/implementations/
+lang: en
+type_of_guidance: false
+footer: ""
+github:
+  repository: w3c/wcag-act-rules
+  path: content/implementations.md
+---
+
+{%- assign siteData = site.data.wcag-act-rules %}
+{%- assign implementations = siteData.act-implementations | sort: "name" %}
+
+<section class="doc-note-box act-sticky">
+  <p>This page is <strong>under development</strong> and has not been approved by the working group.</p>
+</section>
+
+{::nomarkdown} {% include box.html type="start" title="Summary" %} {:/}
+ACT rules provide a standard way to compare the coverage of accessibility test tools and methodologies. This pages shows how many rules different tools and methodologies have accurately implemented. Scores are based on public test data.
+{::nomarkdown} {% include box.html type="end" %} {:/}
+
+## Test Methodologies
+
+Test methodologies provide step by step instructions on how to test accessibility standards.
+See [understanding ACT consistency](#understandingactconsistency).
+
+{% assign methodologies = implementations | where: "type", "Test methodology" %}
+{% include_relative _implementations-table.html implementations=methodologies %}
+
+## Semi-automatic Test Tools
```

Comment:
Should not this be


```suggestion
## Semi-automated Test Tools
```

Reply:
Semi-automated (though not sure that's the best term?)

Reply:
I would say "semi-automated" and "automated" are the best terms in this context, and I would advocate for adopting those. In the preview draft there is still "automatic" and "semi-automatic".